runtime/take_steps.ll

```diff
@@ -8,14 +8,20 @@ declare %block* @step(%block*)
 declare void @finish_rewriting(%block*) #0
 declare void @koreCollect(%block **)
```

Comment:
Why are we writing these files directly in llvm instead of using C++ with C linkage?

Reply:
Just the decision that was made at the time, no particular reason. We could convert them all at a later date, but that should be a separate PR.